categories/views.py

```diff
@@ -1,3 +1,20 @@
-from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View 
+
+from categories.models       import FirstCategory
+
+class CategoryView(View): 
+    def get(self, request):
+        first_categories = FirstCategory.objects.all()
+        result = []
+
+        for first_category in first_categories:
+            second_categories = first_category.second_categories.all()
+            result.append({
+                'first_category_id': first_category.id,
+                'title'            : first_category.title,
+                'second_categories': [{ 'second_category_id' : second_category.id, 'title' : second_category.title } for second_category in second_categories]})
```

Comment:
```suggestion
                'second_categories': [{ 
                    'second_category_id' : second_category.id, 
                    'title'              : second_category.title 
                } for second_category in first_category.second_categories.all() ]
            })
```
질문에 대한 답변으로 다음과 같이 줄바꿈으로 좀 더 가독성을 높일 수 있을 것 같아요!

Reply:
넵 줄바꿈해서 수정했습니다!